### PR TITLE
Component: InfoCard Base PR

### PR DESCRIPTION
**Pull request by TylerEikenberg**
This component is going to be the base component for many other card components. #146 
Changes size and color based on props.

### CODE REVIEW

File: client/src/components/InfoCard/InfoCard.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  width: 100%;
+  min-width: 225px;
+  height: 155px;
+}
+
+.small {
+  width: 100%;
+  min-width: 225px;
+  height: 125px;
+}
```
Comment (ZakkMan) [inline client/src/components/InfoCard/InfoCard.css]: Can you namespace these? i.e. `info-card-small`

File: client/src/components/InfoCard/InfoCard.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./InfoCard.css";
+
+// Function based React Component
+const InfoCard = props => {
+  // Default Class to apply to Component
+  let className = `info-card-container ${props.variant} ${props.size}`;
```
Comment (ZakkMan) [inline client/src/components/InfoCard/InfoCard.js]: Can you do this in the `className` attribute and not create an extra variable?

File: client/src/components/InfoCard/InfoCard.stories.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import InfoCard from "./InfoCard";
+import image from "../../../../assets/images/card_medium1.png"; //here for example
+
+storiesOf("InfoCard", module)
+  .add("Info Card, Purple", () => <InfoCard variant="purple" size="large" />)
```
Comment (ZakkMan) [inline client/src/components/InfoCard/InfoCard.stories.js]: `color` might be a better prop name than `variant`

File: client/src/components/InfoCard/InfoCard.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./InfoCard.css";
+
+// Function based React Component
+const InfoCard = props => {
```
Comment (ZakkMan) [inline client/src/components/InfoCard/InfoCard.js]: Should this just be called `Card`?

**Comment by TylerEikenberg** [comment]
@ZakkMan added all of your suggestions, let me know what you think!

File: client/src/components/InfoCard/InfoCard.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./InfoCard.css";
+
+const Card = props => {
+  let { size = "large", color, picture } = props;
+  return (
+    <div className={`info-card-container info-card-${color} info-card-${size}`}>
+      {picture ? <img src={picture} alt="whoops"></img> : null}
```
Comment (ZakkMan) [inline client/src/components/InfoCard/InfoCard.js]: Could we have the `alt` text passed in as a prop?

Comment (TylerEikenberg) [reply client/src/components/InfoCard/InfoCard.js]: Changed file names and component name to card

**Comment by TylerEikenberg** [comment]
@ZakkMan added the changes you suggested

**Comment by TylerEikenberg** [comment]
@ZakkMan fixed classnames

**Pull request merged**